src/tablecloth/time/utils/validatable.clj

```diff
@@ -22,7 +30,8 @@
                                               (get name))]
     (->> column-names
          (every? (fn [column-name]
-                   (identical? (ds column-name)
-                               (columns column-name)))))
+                   (and (identical? (ds column-name)
+                                    (columns column-name))
+                        (some? (columns column-name))))))
```

Comment:
The `some?` expression here checks that there is a column by `column-name` in `columns`, which is the set of columns that the validatable validates. Really probably `add-validatable` should error when a validatable is added for a column that does not exist.